examples/tensorflow/segmentation/evaluation.py

```diff
@@ -117,7 +117,7 @@ def load_checkpoint(checkpoint, ckpt_path):
 
     if not path_to_checkpoint:
         logger.info('No checkpoint detected')
-        return 0
+        return -1
```

Comment:
What do you think about adding a runtime error here?

Reply:
That's a good point. Returning the runtime error would be more clear. I'll apply it in the code.